### PR TITLE
Change test names to Camel case

### PR DESCRIPTION
**Pull request by odzeno**
Hey, hope I did everything okay :D

### CODE REVIEW

**Comment by odzeno** [comment]
issue #290 
anyone knows why the Travis build failed?

**Comment by stkent** [comment]
Wow, er, that Travis log does not help me much! Looks very odd. @jtigger any ideas?

**Comment by stkent** [comment]
I restarted the build just to see what happens. Science!

**Comment by odzeno** [comment]
haha, of course it works a second time :D

**Comment by stkent** [comment]
Haha, yeah, could have been a Travis instability? Anyway, glad that resolved it 😄

File: exercises/meetup/src/test/java/MeetupTest.java [hunk 1]
```diff
@@ -10,727 +10,727 @@
...
+    public void testSecondWednesdayOfAugust2013() {
         LocalDate expected = LocalDate.of(2013, 8, 14);
         Meetup meetup = new Meetup(8, 2013);
         assertEquals(expected, meetup.day(DayOfWeek.WEDNESDAY, MeetupSchedule.SECOND));
     }
 
     @Ignore
     @Test
-    public void test_second_thursday_of_september_2013() {
+    public void testSeconThursdayOfSeptember2013() {
```
Comment (stkent) [inline exercises/meetup/src/test/java/MeetupTest.java]: Oops, a small typo slipped in here: `Secon` should be `Second`!

Comment (odzeno) [inline exercises/meetup/src/test/java/MeetupTest.java]: oops, will push fix in a minute (:

**Comment by stkent** [comment]
Great, thanks!

**Comment by stkent** [comment]
Nice work, @odzeno!

**Comment by odzeno** [comment]
thanks :D

**Pull request merged**